tests/cli/test_remote.py

```diff
@@ -402,13 +398,11 @@ def test_puttext(remote: Remote, tmp_path: Path):
     dest = tmp_path / f"{dest_dir}/bob.txt"
     some_text = "foo"
     _result = remote.puttext(some_text, str(dest))
-    remote.connection.run.assert_called_once_with(
-        f"mkdir -p {dest_dir}",
-        asynchronous=False,
-        out_stream=None,
-        hide=True,
-        warn=False,
-    )
+    remote.connection.run.assert_called_once()
+    if sys.version_info < (3, 8):
+        assert remote.connection.run.mock_calls[0][1] == (f"mkdir -p {dest_dir}",)
+    else:
+        assert remote.connection.run.mock_calls[0].args[0] == f"mkdir -p {dest_dir}"
```

Comment:
should we start considering dropping support for python 3.8?

Reply:
Yes I agree. I'll make another PR for that.